_posts/2019-07-26-iis-rewrite-maps-and-redirecting-on-url-paths.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: "IIS Rewrite Maps and Redirecting On Url Paths"
+date: 2019-07-26 11:45:20
+tags:
+  - IIS
+  - Azure
+  - SEO
+categories:
+  - IIS
+  - Azure
+  - SEO
+twitter_text:
+authors:
+  - Khalid Abuhakmeh
+  - Kevin Hougasian
+  - Bill Boga
+image: https://images.unsplash.com/photo-1452567489037-30d54d84ad60?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2581&q=80
+image_url: https://unsplash.com/photos/PQvRco_SnpI
+image_credit: darylgio agoncillo
+---
+
+URLs are the bridges that connect our little island of a website to the bigger continent known as the Internet. Preserving URL integrity is the utmost importance for providing an excellent experience for new and existing users alike. Sadly, from a development and management standpoint, keeping old paths working for a new site can be a real pain in the butt. In this post, we'll show you how to use redirect maps to create a lookup dictionary for old URL paths. This post will also help you preserve query string parameters.
+
+## Configuration
+
+We started by exploring the [Microsoft documentation site](https://docs.microsoft.com/en-us/iis/extensions/url-rewrite-module/using-rewrite-maps-in-url-rewrite-module) around rewrite maps. For rewrite maps, you will need to make sure you are running your site on Internet Information Services with the rewrite module.
+
+What you will in your web.config is a `rewriteMaps` section with all off your old URL paths and their new desitations.
```

Comment:
Could you clarify *exactly* how to add these code sections in configuration? Basically, include `system.webServer`, `rules`, etc.